.github/pr-title-checker-config.json

```diff
@@ -4,7 +4,7 @@
     "color": "EEEEEE"
   },
   "CHECKS": {
-    "prefixes": ["fix: ", "feat: ", "feature: ", "chore: ", "hotfix: ", "e2e: "]
+    "prefixes": ["fix: ", "feat: ", "feature: ", "chore: ", "hotfix: ", "e2e: ", "fix!: ", "feat!: ", "feature!: ", "chore!: ", "hotfix!: "]
```

Comment:
I want to inform you that the exclamation syntax flavor is not supported by the release notes generation script. You still can use it, but PRs with such titles will be treated as _Other_.